CustomHeaderPlugin.inc.php

```diff
@@ -83,6 +80,7 @@ function manage($args, $request) {
 
 				AppLocale::requireComponents(LOCALE_COMPONENT_APP_COMMON,  LOCALE_COMPONENT_PKP_MANAGER);
 				$templateMgr = TemplateManager::getManager($request);
+				$templateMgr->registerPlugin('function', 'plugin_url', array($this, 'smartyPluginUrl'));
```

Comment:
Are you sure this is used in this plugin?

Reply:
When I updated this plugin through OJS and activated it, it brought an old version. In that version `register_function` is used. I just noticed that in this repository you first [updated](https://github.com/asmecher/customHeader/commit/2d6ece3867597cbdc91cc99f6c387870b21fb8f7) it and then [removed](https://github.com/asmecher/customHeader/commit/04c7cee122135a719152f0f60ccbfded5c5ba7f5) it. As I applied my OJS changes to the plugin, I missed this line. I will update it. Thanks for pointing it out.